app/logic/Calendar/OWA/OWAEvent.ts

```diff
@@ -330,3 +331,12 @@ function addParticipants(attendees, participants: Participant[]) {
 function extractWeekdays(daysOfWeek: string): Weekday[] | null {
   return daysOfWeek ? daysOfWeek.split(" ").map(day => sanitize.integer(Weekday[day])) : null;
 }
+
+function fromWindowsZone(windowsZone): string | null {
+  for (let iana in WindowsTimezones) {
```

Comment:
Does OWA use only Windows timezones? Not IANA timezones at all?

Would be nice to also support IANA timezones. In the off chance that Microsoft decides one day to use standards. Or so...

Reply:
Same for the other protocols.

Reply:
I can't prove it for sure but the list of timezones in the OWA UI does seem to match the list of Windows zones. Neither the EWS nor the ActiveSync documentation is very helpful.